core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -66,26 +65,31 @@ class MessageTest extends JUnitSuite {
       } else {
         TestUtils.checkEquals(ByteBuffer.wrap(v.payload), v.message.payload)
       }
+
       // check timestamp
-      if (v.magicValue > Message.MagicValue_V0)
+      if (v.magicValue > Message.MagicValue_V0) {
         assertEquals("Timestamp should be the same", v.timestamp, v.message.timestamp)
-       else
+      } else {
         assertEquals("Timestamp should be the NoTimestamp", Message.NoTimestamp, v.message.timestamp)
+      }
```

Comment:
- Lucky you, [Apple](https://www.imperialviolet.org/2014/02/22/applebug.html) would like to be in your shoes :)
- It does not have to be bug, imagine to debug the code above and for a while you would like to comment out `somethingElse()`
- I would argue (and majority of java community) that second example is more readable and maintainable. To offer similar argument like you, I have not met any developer who would write `if` without curly braces.